rialto_airflow/dags/harvest.py

```diff
@@ -29,6 +29,15 @@
 except ValueError:
     pass
 
+if dev_limit is None:
+    logging.info(
+        f"⚠️ dev_limit is set to {dev_limit}, running harvest will stop at the limit number of publications per source"
+    )
+else:
+    logging.info(
+        "‼️ no dev_limit is set, running harvest will attempt to retrieve all results"
+    )
```

Comment:
i find this sort of thing useful, happy to remove if others think it's more noise than signal